### DISCUSSION TITLE
What is item_id mean in TimeSeriesDataFrame?

### DISCUSSION THREAD

**Discussion by ygbingo** [amazon-science/chronos-forecasting#175] [Q&A]
I saw a code:
```python
train_data = TimeSeriesDataFrame.from_data_frame(
    df,
    id_column="item_id",
    timestamp_column="timestamp"
)
train_data.head()
```
and doc says: unique ID of the time series ("item_id") as int or str. But I still don't know what's the id_column's mean? Could anybody give another answer?

**Comment by lostella** [comment]
Hi @ygbingo, the `item_id` is any ID that allows you to identify the series. The predictions dataframe will be indexed using the same ID information, so that you can easily know which forecasts are for what.

Please note that this is a feature of [autogluon](https://github.com/autogluon/autogluon), and not the current repository. You can take a look at the [tutorials here](https://auto.gluon.ai/stable/tutorials/timeseries/index.html) to know more about autogluon's forecasting features.
